src/extensions/addrbar/bootstrap-table-addrbar.js

```diff
@@ -44,6 +44,8 @@ function _buildUrl (dict, url = window.location.search) {
     // 搜索name=general这种形式的字符串(&是分隔符)
     const pattern = `${key}=([^&]*)`
     const targetStr = `${key}=${val}`
+    if (targetStr.includes('undefined'))
```

Comment:
`typeof targetStr === undefined`

Reply:
This don't works because targetStr is string like this
```
sort=undefined
```

Reply:
oh your right, then `val === undefined` should work i guess.

Reply:
Ok, I change this line.